engines/bladerunner/POTFILES

```diff
@@ -1 +1,2 @@
 engines/bladerunner/bladerunner.cpp
+engines/bladerunner/detection.cpp
```

Comment:
You are missing a end of line on this last line.
Our build system does not like that and would concatenate the line with the first line of the next POTFILES to be processed, resulting in an error.